frontend/public/components/machine-config-pool.tsx

```diff
@@ -37,7 +39,13 @@ const pauseAction = (kind, obj) => ({
 });
 
 const machineConfigPoolReference = referenceForModel(MachineConfigPoolModel);
+const nodeReference = referenceForModel(NodeModel);
 const machineConfigPoolMenuActions = [pauseAction, ...Kebab.factory.common];
+const hasCondition = (mcp: MachineConfigPoolKind, type: MachineConfigPoolConditionType): K8sResourceConditionStatus => {
```

Comment:
I'd probably call this `conditionStatus`. It sounds like it returns a boolean, but it returns a string

Reply:
Fixed